c7n/resources/asg.py

```diff
@@ -1761,3 +1761,80 @@ def process_config(self, client, config):
             if e.response['Error']['Code'] == 'ValidationError':
                 return
             raise
+
+
+@resources.register('scaling-policy')
+class ScalingPolicy(query.QueryResourceManager):
+
+    class resource_type(query.TypeInfo):
+        service = 'autoscaling'
+        arn_type = "scalingPolicy"
+        id = name = 'PolicyName'
+        date = 'CreatedTime'
+        enum_spec = (
+            'describe_policies', 'ScalingPolicies', None
+        )
+        filter_name = 'PolicyNames'
+        filter_type = 'list'
+        cfn_type = config_type = 'AWS::AutoScaling::ScalingPolicy'
```

Comment:
auto scaling isn't supported by config so config_type should be removed here, this is currently triggering a test failure.

Reply:
Done